.test-infra/jenkins/job_PostCommit_Java_Nexmark_Spark.groovy

```diff
@@ -23,175 +23,16 @@ import CommonTestProperties.TriggeringContext
-                '--streamTimeout=60' ,
-                '--manageResources=false',
-                '--monitorJobs=true'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK SQL IN BATCH MODE USING SPARK 3 RUNNER ***"')
-        gradle {
-          rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:nexmark:run')
-          commonJobProperties.setGradleSwitches(delegate)
-          switches('-Pnexmark.runner=":runners:spark:3"' +
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=SparkRunner',
-                '--queryLanguage=sql',
-                '--streaming=false',
-                '--suite=SMOKE',
-                '--streamTimeout=60' ,
-                '--manageResources=false',
-                '--monitorJobs=true'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK IN BATCH MODE USING SPARK 2 STRUCTURED STREAMING RUNNER ***"')
-        gradle {
-          rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:nexmark:run')
-          commonJobProperties.setGradleSwitches(delegate)
-          switches('-Pnexmark.runner=":runners:spark:2"' +
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=SparkStructuredStreamingRunner',
-                '--streaming=false',
-                '--suite=SMOKE',
-                // Skip query 3 (SparkStructuredStreamingRunner does not support State/Timers yet)
-                '--skipQueries=3',
-                '--streamTimeout=60' ,
-                '--manageResources=false',
-                '--monitorJobs=true'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK SQL IN BATCH MODE USING SPARK 2 STRUCTURED STREAMING RUNNER ***"')
-        gradle {
-          rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':sdks:java:testing:nexmark:run')
-          commonJobProperties.setGradleSwitches(delegate)
-          switches('-Pnexmark.runner=":runners:spark:2"' +
-              ' -Pnexmark.args="' +
-              [
-                commonJobProperties.mapToArgString(nexmarkBigQueryArgs),
-                commonJobProperties.mapToArgString(nexmarkInfluxDBArgs),
-                '--runner=SparkStructuredStreamingRunner',
-                '--queryLanguage=sql',
-                '--streaming=false',
-                '--suite=SMOKE',
-                '--streamTimeout=60' ,
-                '--manageResources=false',
-                '--monitorJobs=true'
-              ].join(' '))
-        }
-        shell('echo "*** RUN NEXMARK IN BATCH MODE USING SPARK 3 STRUCTURED STREAMING RUNNER ***"')
```

Comment:
Missing spark 3 tests. Also, are they being treated the same metric as spark 2?

Reply:
Yes, I fixed both issues. The new metric is `SparkStructuredStreamingRunner3`. Looks like this metric duplication issue has been around since #15529.

Reply:
This is actually going to fail when it runs. I'll probably open a bug and fix in a followup.

Reply:
Filed #23634